Documentation/api.md

```diff
@@ -0,0 +1,37 @@
+#### Api endpoint for V2
+
+There are two endpoints for the V2 versions of the endpoint:
+-`/api/v2/reports/{name}/full`
+-`/api/v2/reports/{name}/table`
+
+These are located in pkg/chargeback/http.go.
+
+{name} is the name if the report that you are looking to run. Formatting is specified as a query string at the end.
+
+#### Sample urls
+
+Replace namespaceVar with your namespace
+Replace reportnameVar with the name of your report
+Replace formatVar with json, csv or tabular
+
+#Full endpoint
+```
+$ curl "http://127.0.0.1:8001/api/v1/namespaces/namespaceVar/services/metering:http/proxy/api/v2/reports/reportnameVar/full?format=formatVar"
+```
+#Table endpoint
+```
+$ curl  "http://127.0.0.1:8001/api/v1/namespaces/namespaceVar/services/metering:http/proxy/api/v2/reports/reportnameVar/table?format=formatVar"
+
+```
+
+The endpoint `/api/v2/reports/{name}/full` performs a similar function to the `/api/v1/reports/get` endpoint, except the difference is in the formatting of the output display.
+
+Sample Output in JSON:
+ {"results":[{"values":[{"name":"data_start","value":"2018-08-13T20:35:00Z","tableHidden":false,"unit":"date"},{"name":"data_end","value":"2018-08-13T23:58:00Z","tableHidden":false,"unit":"date"},{"name":"pod_request_cpu_core_seconds","value":2412,"tableHidden":false,"unit":"cpu_core_seconds"},{"name":"period_start","value":"2018-01-01T00:00:00Z","tableHidden":false,"unit":"date"},{"name":"period_end","value":"2018-12-30T23:59:59Z","tableHidden":false,"unit":"date"},{"name":"namespace","value":"default","tableHidden":false,"unit":"kubernetes_namespace"}]},
+
+ The endpoint `/api/v2/reports/{name}/table`
+
+ tableHidden is a column in the reportqueries that allows Metering users to fine tune what cloumns are shown or hidden in their reports. tableHidden takes a bool and shows/hides columns where this is set. Only where the bool is set to false will the column return.
```

Comment:
`tableHidden` isn't a column, it's a field on the Column in a ReportGeneration query.

Reply:
`cloumns` mispelling.

Reply:
>  tableHidden takes a bool and shows/hides columns where this is set. Only where the bool is set to false will the column return.

could be reworded to something like:

>  tableHidden is a boolean and controls if a column should be shown when displayed in a table. If it's true, then the `/api/v2/reports/{name}/table` endpoint will omit this column and it's values from the response (in all formats).